crates/solver/src/solver/oneinch_solver.rs

```diff
@@ -129,11 +129,6 @@ impl OneInchSolver {
         tracing::debug!("querying 1Inch swap api with {:?}", query);
         let swap = match self.client.get_swap(query).await {
```

Comment:
I think this can now be simplified to:
```suggestion
        let swap = match self.client.get_swap(query).await?;
```